tests/jballerina-unit-test/src/test/resources/test-src/statements/clientdeclstmt/client_decl_stmt_annotations_negative_test.bal

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+const annotation record {| int i; |} A5 on source client;
```

Comment:
Lests add `INVALID_NON_ANYDATA_CLIENT_DECL_ANNOTATION` tests for statement level as well

Reply:
This validation is done for the annotation declaration which is only allowed at module-level.